extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -60,13 +60,22 @@ describes.realWin(
 
     async function shoppingDataDispatchStoreService() {
       const shoppingData = {
-        'sunglasses': {'productTitle': 'Spectacular Spectacles'},
+        'page1': {'sunglasses': {'productTitle': 'Spectacular Spectacles'}},
       };
       storeService.dispatch(Action.ADD_SHOPPING_DATA, shoppingData);
     }
 
-    it('should build and layout shopping tag component', () => {
+    async function createShoppingTag() {
+      shoppingTag.element.setAttribute('data-product-id', 'sunglasses');
+      await shoppingDataDispatchStoreService();
+
+      expect(() => shoppingTag.buildCallback()).to.not.throw();
       expect(() => shoppingTag.layoutCallback()).to.not.throw();
+    }
```

Comment:
There's already a `createAmpStoryShoppingTag()` function, which makes the existence of this new function confusing, so you should just move the contents of this function into your new test below. By the way, I think we only tend to call `expect()` within tests and not helper functions